apk-generator/v2/scripts/build_apk.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
-./gradlew assembleRelease --info
+./gradlew assemble$3Release --info
```

Comment:
Apologies ... My bad 😅 ... Thank you for pointing it out @Shailesh351 ... @arp95 please do the above changes.